db/migrations/20240514205117-trim-tabs-from-plan-names.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+exports.up = async (q) => {
+    // manually update the following non-compliant remediation plan names:
+    const updates = {
+        "133bd18f-7134-4a83-8034-77ba15ca8327": "A newer version of kernel is installed: but not used. Bug fixes issued by this new kernel version are not applied",
```

Comment:
Instead of writing down concrete ids from certain environment(s), I'd suggest to do the change programmatically to handle all environments consistently. That would also avoid leaking data into application's git repository.

Reply:
While I agree, sequelize seems to want to sanitize parts of the SQL and won't let me pass a tab character to the TRIM() function.  I've already spent 6 hours trying to debug sequelize migration processing and just need to get this done since it's leading to pod restarts in prod.  If I had access to a db console I would just fix these by hand but that's not allowed, so here we are.

Reply:
Yeah, 🤮...  After _much_ hacking I found a gross workaround, but it's better than targeting specific ids.